main/src/ca/uwaterloo/flix/language/ast/NamedAst.scala

```diff
@@ -172,6 +172,18 @@ object NamedAst {
 
     case class NativeMethod(method: Method, args: List[NamedAst.Expression], tpe: ast.Type.Var, loc: SourceLocation) extends NamedAst.Expression
 
+    case class NewChannel(tpe: NamedAst.Type, loc: SourceLocation) extends NamedAst.Expression
```

Comment:
NewChannel will still need a tpe: ast.Type.Var in NamedAst as this is used in the next phase.